docs/learn/dapp-developer/read-profile-data.md

```diff
@@ -144,154 +134,197 @@ You can give it a try with this profile address: [`<myProfileAddress> = 0xE1F684
+  "0x03B2689E4843ca56B2A933e7eC1E1ee6C3e6982e",
+  "https://rpc.testnet.lukso.network",
+  {
+  ipfsGateway: "https://api.universalprofile.cloud/ipfs",
+  }
+);
+
+  const specificProfileData = await erc725js.getData([
+    "LSP3Profile",
+    "LSP1UniversalReceiverDelegate",
+  ]);
 ```
 
+</details>
+
 <details>
     <summary>Show result</summary>
 
 ```js
-{
-  "key": "0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5",
-  "name": "LSP3Profile",
-  "value": {
-    "LSP3Profile": {
-      "name": "johann",
-      "description": "I'm a 40 y-old dad of 3. Technology enthusiast, skater, guitarist but mostly curious.",
-      "tags": [
-        "profile"
-      ],
-      "links": [
-        {
-          "title": "...",
-          "url": "..."
-        },
-        ...
-      ],
-      "profileImage": [
-        {
-          "width": 1512,
-          "height": 1998,
-          "hashFunction": "keccak256(bytes)",
-          "hash": "0x...",
-          "url": "ipfs://..."
-        },
-        ...
-      ],
-      "backgroundImage": [
-        {
-          "width": 1512,
-          "height": 1998,
-          "hashFunction": "keccak256(bytes)",
-          "hash": "0x...",
-          "url": "ipfs://..."
-        },
-        ...
-      ]
-    }
-  }
-}
+[
+  {
+    key: '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
+    name: 'LSP3Profile',
+    value: {
+      verification: {
+        method: 'keccak256(utf8)',
+        data: '0x598c2707d6bd256489e5de43f60f532f6541c6785c94f85a348e9bcbd7bdf4d6',
+      },
+      url: 'ipfs://QmPNk4GXUDVSpkMYS9ySLj4r7WJYNBLG986GFPqfRJPL8E',
+    },
+  },
+  {
+    key: '0x0cfc51aec37c55a4d0b1a65c6255c4bf2fbdf6277f3cc0730c45b828b6db8b47',
+    name: 'LSP1UniversalReceiverDelegate',
+    value: '0x7870C5B8BC9572A8001C3f96f7ff59961B23500D',
+  },
+];
 ```
 
 </details>
 
-:::note get and fetch
+## Retrieve decoded data
 
-The [`getData(...)`](../../tools/erc725js/classes/ERC725#getdata) function only retrieves the data keys values from the smart contract. In comparison, [`fetchData(...)`](../../tools/erc725js/classes/ERC725#fetchdata) will download and decode the content of `VerifiableURI`.
+#### Get all profile metadata (name, photos...)
 
-:::
+Example to retrieve the content of the JSON file from the verifiable URI stored on the smart contract. It will download the JSON file and verify its hash automatically.
 
-## Fetch Assets and Universal Receiver
+<details>
+    <summary>Show code snippet</summary>
 
-Instead of using the [`LSP3Profile`](../../standards/universal-profile/lsp3-profile-metadata) key, you can also use other data keys like [`LSP12IssuedAssets[]`](../../standards/universal-profile/lsp12-issued-assets), [`LSP5ReceivedAssets[]`](../../standards/universal-profile/lsp5-received-assets), or [`LSP1UniversalReceiverDelegate`](../../standards/generic-standards/lsp1-universal-receiver-delegate), as described in the following example:
+```js
```

Comment:
does the syntax highlighter supports `ts` ? The code below is TS so might be more relvant to put ts instead if it is supported